doc/js_tutorials/js_setup/js_nodejs/js_nodejs.markdown

```diff
@@ -231,8 +231,57 @@ In the browser, this filesystem is emulated in memory while in Node.js there's a
 const { Canvas, createCanvas, Image, ImageData, loadImage } = require('canvas');
 const { JSDOM } = require('jsdom');
 const { writeFileSync, existsSync, mkdirSync } = require('fs');
+const https = require('https');
 
 (async () => {
+const createFileFromUrl = function (path, url, maxRedirects = 10) {
+  console.log('Downloading ' + url + '...');
+  return new Promise((resolve, reject) => {
+    const download = (url, redirectCount) => {
+      if (redirectCount > maxRedirects) {
+        reject(new Error('Too many redirects'));
+      } else {
+        https.get(url, (response) => {
+          if (response.statusCode === 200) {
+            let data = [];
+            response.on('data', (chunk) => {
+              data.push(chunk);
+            });
+
+            response.on('end', () => {
+              try {
+                writeFileSync(path, Buffer.concat(data));
+                resolve();
+              } catch (err) {
+                reject(new Error('Failed to write file ' + path));
+              }
+            });
+          } else if (response.statusCode === 302 || response.statusCode === 301) {
+            download(response.headers.location, redirectCount + 1);
+          } else {
+            reject(new Error('Failed to load ' + url + ' status: ' + response.statusCode));
+          }
+        }).on('error', (err) => {
+          reject(new Error('Network Error: ' + err.message));
+        });
+      }
+    };
+    download(url, 0);
+  });
+};
+
+if (!existsSync('./face_detection_yunet_2023mar.onnx')) {
+  await createFileFromUrl('./face_detection_yunet_2023mar.onnx', 'https://media.githubusercontent.com/media/opencv/opencv_zoo/main/models/face_detection_yunet/face_detection_yunet_2023mar.onnx')
+}
+
+if (!existsSync('./opencv.js')) {
+  await createFileFromUrl('./opencv.js', 'https://docs.opencv.org/master/opencv.js')
```

Comment:
please use 5.x branch, but not master. master is shadow copy of 4.x and may be removed soon.

Reply:
Done.